### ISSUE TITLE
Installing danger via OS X system Ruby does not work correctly

### ISSUE THREAD

**Issue by neonichu** [danger/danger#260]
```bash
$ sudo gem install danger
1 gem installed
$ danger
/usr/local/bin/danger:23:in `load': cannot load such file -- /Library/Ruby/Gems/2.0.0/gems/danger-0.8.1/bin/danger (LoadError)
	from /usr/local/bin/danger:23:in `<main>'
$ ls -ld /Library/Ruby/Gems/2.0.0/gems/danger-0.8.1/bin/danger
-rwx--x--x  1 root  wheel  118 Jun  8 23:04 /Library/Ruby/Gems/2.0.0/gems/danger-0.8.1/bin/danger
```

The files are all there, but none of them has read access for group/other — I'm thinking something went wrong when packaging, because the permissions on the files in the repository itself look fine.

**Comment by orta** [comment]
weird, I use the default rake tasks for handing gems

**Comment by segiddins** [comment]
@orta that take task won't handle permissions. Are you sure ruby gems didn't emit any errors while packaging?

**Comment by orta** [comment]
about to do a new release, will check

**Comment by segiddins** [comment]
do a `gem build danger.gemspec` first to see the warnings before releasing

**Comment by orta** [comment]
heh, yep - all broke

![screen shot 2016-06-09 at 12 38 46 pm](https://cloud.githubusercontent.com/assets/49038/15937994/21614e34-2e3f-11e6-9b4e-45c070c73307.png)

**Comment by orta** [comment]
Looks like all the rake tasks hide this stuff ^ - wouldn't have noticed without that - thanks @segiddins 

This is fixed in 0.8.2

For people wanting to figure it out ( wasn't too many resources ) I ran this in pry:

``` pry
 Dir[File.join('{.,bin,assets,lib,spec,tasks}', '**/*')].each do |file|
    mode = File.directory?(file) ? 0755 : 0644
    File.chmod(mode, file)
  end
```

**Comment by orta** [comment]
For myself in the future - I now use `chmod -R a+r .` to fix this
